### DISCUSSION TITLE
Duplicate q-item lists

### DISCUSSION THREAD

**Discussion by dost-mmg** [quasarframework/quasar#13215] [General - Components / Directives / etc] [answered]
I am trying to create a q-select list using options assign to an array with object values. I would like the label to be one of the values, and the value assigned to the selection to be the other value. I've gotten it working, but the problem is that instead of a q-item with two items, I get a list with four items -- they are duplicated. 

Here is a pen https://codepen.io/kiggs1881/pen/KKZWwBw

**Comment by MilosPaunovic** [comment, marked as answer]
[https://codepen.io/milospaunovic/pen/RwxvGEz?editors=1111](https://codepen.io/milospaunovic/pen/RwxvGEz?editors=1111)

**Reply by dost-mmg** [reply]
It works! Of course. Thanks always for your work and time.
